### PR TITLE
ascanrulesBeta: Add more example alerts

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Add notes.
- Scan rules > Add example alert functionality (6119).
- Unit tests > Assert the new example alerts.
- Messages.properties > Updated some http references (8262).

## Related Issues
- zaproxy/zaproxy#6119
- zaproxy/zaproxy#8262

## Checklist
- [NA] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java [hunk 1]
```diff
@@ -661,4 +665,15 @@ public int getWascId() {
...
     }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+                buildAlert(
+                                "https://example.com/profile/ybpsv/bqmmn/?foo=bar",
+                                Constant.messages.getString(
+                                        MESSAGE_PREFIX + "extrainfo.nocontenttype"),
+                                "background: url(image.png)")
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java:676]: This is based on my reading of the code, please dbl check.

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/IntegerOverflowScanRule.java [hunk 2]
```diff
@@ -221,4 +221,27 @@ private boolean attackVector(String param, char type, String returnAttack) {
...
+                                "https://example.com/?years=1",
+                                "years",
+                                "95697568703220167658153205694899573480013738",
+                                '1',
+                                "HTTP/1.1 500 Internal Server Error")
+                        .build());
+    }
+
+    private AlertBuilder buildAlert(
+            String url, String param, String returnAttack, char type, String evidence) {
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/IntegerOverflowScanRule.java:238]: `attack`

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java [hunk 3]
```diff
@@ -642,6 +637,15 @@ public void scan() {
         }
     }
 
+    private AlertBuilder buildAlert(
+            String attack, String extraInfo, String relativeReferenceEvidence) {
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java:641]: `otherInfo`, `evidence`

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**